sklearn/preprocessing/tests/test_data.py

```diff
@@ -1217,7 +1218,7 @@ def test_normalize():
 
 
 def test_binarizer():
-    X_ = np.array([[1, 0, 5], [2, 3, -1]])
+    X_ = np.array([[1, 0, 5], [2, 3, -1]], dtype=np.float64)
```

Comment:
Why was the dtype conversion done here (https://github.com/scikit-learn/scikit-learn/pull/5398/files#diff-5ebddebc20987b6125fffc893f5abc4cR1336) in the first place, I removed it and all tests seem to pass.